docs/source/markdown/podman-generate-systemd.1.md

```diff
@@ -94,9 +94,108 @@ PIDFile=/run/user/1000/overlay-containers/ccfd5c71a088768774ca7bd05888d55cc28769
 WantedBy=multi-user.target default.target
 ```
 
+Installation of generated systemd unit files.
+
+Podman generated unit files include an `[Install]` section, which carries installation information for the unit. It is used by the enable and disable commands of the systemctl(1) during installation.
+
+Once you have generated the systemd unit file. Copy the generated systemd file to ```/usr/lib/systemd/system``` for installing as a root user and to ```/usr/lib/systemd/user``` for installing it as a non-root user.
+
+Enable the copied one or more units. This will create a set of symlinks, as encoded in the "[Install]" sections of the indicated unit files.
+
+Note: Coping the unit to ```/usr/lib/systemd/system``` and enabling it marks the unit to be automatically started at boot.
```

Comment:
```suggestion
Note: Coping a unit to ```/usr/lib/systemd/system``` and enabling it marks the unit to be automatically started at boot.
```

Reply:
What about user services? Those require lingering, and I believe that's the right place to mention.

Reply:
I have added the details with enabling lingering and installing unit files as a user. @vrothberg PTAL now.